### PR TITLE
Enable publishing to b7a central

### PR DESCRIPTION
**Pull request by ThishaniLucas**
$subject as well as replacing module with package in MDs.

### CODE REVIEW

**Review by ayeshLK — APPROVED** [review]

**Review by ThisaruGuruge — APPROVED** [review]
LGTM

File: regex-ballerina/build.gradle [hunk 1]
```diff
@@ -212,3 +222,6 @@ publishing {
 
 updateTomlVersions.dependsOn unpackJballerinaTools
 build.dependsOn ballerinaBuild
+
+ballerinaPublish.dependsOn ballerinaBuild
+publish.dependsOn ballerinaPublish
```
Comment (ThisaruGuruge) [inline regex-ballerina/build.gradle:227]: `test` task should be depending on `ballerinaTest`. `ballerinaBuild` should be depending on `test` task.

Comment (ThishaniLucas) [reply regex-ballerina/build.gradle:231]: Thanks for pointing it out. Reverted the deletion of the test task.

**Review by ThisaruGuruge — COMMENTED** [review]
LGTM

**Review by aashikam — APPROVED** [review]

**Pull request merged**